frontend/js/src/settings/delete/DeleteAccount.tsx

```diff
@@ -67,10 +67,11 @@ export default function DeleteAccount() {
       toast.success(
         <ToastMsg
           title="Success"
-          message={`Successfully deleted acount for ${name}.`}
+          message={`Successfully enqueued account deletion for ${name}.`}
         />
       );
 
+      // TODO: logout user properly
```

Comment:
This should probably be done before merging. And also will a user be prevented from logging in again? I bet people will do that "just to check".

Reply:
Yes, this is already broken. I just noticed it when testing the new changes. @MonkeyDo, can you please help with this? I am not sure and will have to lookup the correct way to logout users with react-router-dom.

We aren't preventing users from logging in again. I think we should clarify that in the error message. I did update it to say the account has been enqueued for deletion as opposed to the account has been deleted. But maybe we can improve it more.

Reply:
done

Reply:
Sorry just saw the message now. Do you still need any help?